test/go-lint.sh

```diff
@@ -8,7 +8,13 @@ if [ -n "${unformatted_files}" ]; then
     exit 1;
 fi
 
+rc=0
 for engine in multiuser insecure; do
   echo "Running golangci-lint for ${engine} engine.."
   golangci-lint run --build-tags "${engine}"
+  current_rc=$?
+  if [ $current_rc -ne 0 ]; then
+    rc=$current_rc
+  fi
 done
+exit $rc
```

Comment:
can you not just change line 1 to `#!/bin/bash -e` ? this way, it will fail straight away, so should save some compute time.

Reply:
Wouldn't this fail on the first failure then? I was hoping to run both engines. Either way is fine by me.